func/sender.py

```diff
@@ -29,8 +30,13 @@ def sender( filepath, filename, receiver ):
     part.add_header('Content-Disposition', 'attachment', filepath = os.path.basename(filepath), filename=filename)
     msg.attach( part )
 
-    server = smtplib.SMTP( 'smtp.gmail.com:587' )
-    server.starttls()
-    server.login( username, password )
-    server.sendmail( msg['From'], msg['To'], msg.as_string() )
+    try:
+        server = smtplib.SMTP( 'smtp.gmail.com:587' )
+        server.starttls()
+        server.login( username, password )
+        server.sendmail( msg['From'], msg['To'], msg.as_string() )
+        return 1
+    except:
```

Comment:
Please try to log that error also that u catch in exception

Reply:
Okay. Working on it.